Source/textdat.cpp

```diff
@@ -654,48 +654,102 @@ const TextDataStruct alltext[] = {
 	    1, 5, TSFX_BMAID26 },
-	{ "I know that my grandmother’s story seems strange but the graveyard does have many mysteries surrounding it. It couldn’t hurt you to put something there see what happens, could it? Maybe you will find out something that will help us all."
+	{ "I know that my grandmother’s story seems strange but the graveyard does have many mysteries surrounding it. It couldn’t hurt you to put something there see what happens, could it? Maybe you will find out something that will help us all. |"
 	    1, 5, TSFX_BMAID28 },
-	{ "My grandmother had a dream about you last night, she said that in her dream you used your bare hands to defeat one of those fallen monsters that lurk under the church. When it died it exploded into a fabulous treasure! Although she was a bit frightened by her dream, she was able to describe the beast quite clearly to me."
+	{ "My grandmother had a dream about you last night, she said that in her dream you used your bare hands to defeat one of those fallen monsters that lurk under the church. When it died it exploded into a fabulous treasure! Although she was a bit frightened by her dream, she was able to describe the beast quite clearly to me. |"
 	    1, 5, TSFX_BMAID29 },
-	{ "My grandmother had that exact same dream again. She said you are quite brave and seemed very pleased with your prize."
+	{ "My grandmother had that exact same dream again. She said you are quite brave and seemed very pleased with your prize. |"
 	    1, 5, TSFX_BMAID30 },
-	{ "I really don't know much about the priest Tremain. He never visits the tavern, preferring to keep to the company of Pepin and Cain. Perhaps it is because they, too, have more scholarly pursuits."
+	{ "I really don't know much about the priest Tremain. He never visits the tavern, preferring to keep to the company of Pepin and Cain. Perhaps it is because they, too, have more scholarly pursuits. |"
 	    1, 5, TSFX_BMAID38 },
+	{ "I've seen no such things in my shop, but I will keep a close watch for them. Perhaps if they come this way they won't take kindly to the fires of my forge. |",
+	    1, 5, TSFX_SMITH5 },
+	{ "I admit that I too have seen these strange lights, but I have not felt pulled towards them. I'll be sure to steer clear of them, and I'll tell everyone that visits here to do likewise. |",
+	    1, 5, TSFX_SMITH6 },
+	{ "A bold tale indeed. My limited time beneath the cathedral leaves me poorly equipped to offer you any help of this. But, as always, you'll find Cain well versed in legends and folklore. |",
+	    1, 5, TSFX_SMITH8 },
+	{ "The blade Azurewrath *Griswold noise* It's legend! It was cast by the angelic weaponsmith Cinadide, and tempered within the fires of judgment! Whoever wields this weapon will find the legions of Hell at his feet! If you found this blade, I would begin to truly believe that you could end the nightmare that has befallen our town. |",

```

Comment:
Are the comments like `Griswold Noise` intentionally left in for reference?

Reply:
Oops, nice catch. Copy+paste error from https://diablo-evolution.net/
I will change to "Agggh" instead

Reply:
Griswold noises. It's so unique, it's an accent of its own :P